ublox_derive/src/output.rs

```diff
@@ -227,36 +235,56 @@ pub fn generate_send_code_for_packet(pack_descr: &PackDesc) -> TokenStream {
             pub #name: #ty
         });
         let size_bytes = match f.size_bytes {
-            Some(x) => x,
-            None => unimplemented!(),
+            Some(x) => x.get(),
+            None => 0,
         };
-        if let Some(into_fn) = f.map.map_type.as_ref().map(|x| &x.into_fn) {
-            pack_fields.push(quote! {
-                let bytes = #into_fn(self.#name).to_le_bytes()
-            });
-        } else if !f.is_field_raw_ty_byte_array() {
-            pack_fields.push(quote! {
-                let bytes = self.#name.to_le_bytes()
+
+        if size_bytes == 0 {
```

Comment:
I think we could also flip this if statement, and have:
```
if let Some(size_bytes) = f.size_bytes {
    // everything currently in the else condition and after
    if let Some(into_fn) = .... {
    }
    ...
    off += size_bytes;
} else {
    extend_fields.push(...);
}
```
and then we don't have to map from None to 0, and there isn't that continue splitting up the code.

But, bigger question: because of the `off += size_bytes` that we're skipping (because we don't know the actual size_bytes), will this code break if this field is not the last one?

If so, should we just assert that this is the last field, and break?